src/test/java/org/jenkinsci/plugins/github/pullrequest/GitHubPRRepositoryTest.java

```diff
@@ -97,10 +96,8 @@ public void doClearRepoPullsDeleted() throws IOException {
         PowerMockito.mockStatic(BulkChange.class);
         when(BulkChange.contains(repo)).thenReturn(true);
 
-        FormValidation formValidation = repo.doClearRepo();
-
-        Assert.assertEquals(FormValidation.Kind.OK, formValidation.kind);
-        Assert.assertEquals(null, repo.getPulls());
+        assertThat(repo.doClearRepo().kind, equalTo(FormValidation.Kind.OK));
+        assertThat(repo.getPulls().keySet(), hasSize(0));
```

Comment:
assertThat is more refactorible, because it uses matchers (it writes good verification messages and easily can be changed) and you don't need to remember where the expected value should be (the common problem of lots). One more is that assert that more human readable.